azure-devops/run-build.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       inputs:
         cmakeListsOrSettingsJson: 'CMakeListsTxtAdvanced'
         cmakeListsTxtPath: '$(Build.SourcesDirectory)/CMakeLists.txt'
-        buildDirectory: $(Build.ArtifactStagingDirectory)/${{ parameters.targetPlatform }}
+        buildDirectory: D:/build/${{ parameters.targetPlatform }}

```

Comment:
These changes also appear in #700 - were the PRs mixed?

Also, in that PR I said: I believe `D:/build` should be centralized in the `variables` at the top, like `vcpkgLocation`.

Reply:
Billy suggested I test the changes independently instead of mixing changes.